Sources/Server/Response.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 public class Response {
 
     typealias PathHandler = (Response) -> (Request) -> String
-    var paths: [String: PathHandler] = ["/": handleRoot,"/hello": handleHello,"/coffee": handleCoffee, "/tea": handleRoot, "/parameters": handleParameters, "/cookie": handleCookie, "/eat_cookie": handleEatCookie, "/redirect": handleRedirect, "/form": handleForm]
+    var paths: [String: PathHandler] = ["/": handleRoot,"/hello": handleHello,"/coffee": handleCoffee, "/tea": handleRoot, "/parameters": handleParameters, "/cookie": handleCookie, "/eat_cookie": handleEatCookie, "/redirect": handleRedirect, "/form": handleForm, "/method_options": handleOptions1, "/method_options2": handleOptions2]
```

Comment:
I understand that this will pass cob_spec, but the idea of options is you can send it to any endpoint and the server will tell you the available options. Not sure how hard we want to push for this now, but it might be worth revisiting especially when implementing the `Method Not Allowed` portion. Thoughts @cdemyanovich?

Reply:
Agreed @tommcgee.

Reply:
On the other hand, there's this. :-)
![37 pieces of flair](https://cdn.meme.am/instances/67417093/you-know-what-if-you-want-me-to-wear-37-pieces-of-flair-just-make-the-minimum-37-pieces-of-flair.jpg)